app/views/job_post_requests/new.html.haml

```diff
@@ -4,14 +4,14 @@
 - title 'Job Post Request'
 
 %li#job_post_request
-  %h1 Find Indianapolis' Most Passionate Programmers
+  %h1 Find Indianapolis' Most Passionate Programmers, Designers, PMs, Testers, and More
 
-  %p You want to get your open position in front of quality, local programmers. Lucky for you, this site's demographic consists primarily of high quality programmers that live in the Indianapolis metropolitan area. If you need a nerd, you need to post here.
+  %p You want to get your open position in front of quality, local makers. Lucky for you, this site's demographic consists primarily of high quality makers that live in the Indianapolis metropolitan area. If you need a nerd, you need to post here.
```

Comment:
I chose the word "makers" to use in a couple places because it's shorter than listing out several disciplines. I felt that was an appropriate term, but I'm open to conversation about it.

Reply:
Oh, man. This is definitely a word that we could easily agonize over. In much of our discussion of our mission, we leaned toward "tech people" or people "interested in tech". I think we've delved into discussions of whether a product marketer (or even product manager) is technically a "maker" or not. 

I don't feel strongly, but I'd advocate something like "tech people" to be more inclusive—or at least prevent folks from spending too much time wondering whether they belong. :-)

What say you? Am I taking it too far?

Reply:
Suggested change:

You want to get your open position in front of the local tech community. Lucky for you, Indy Hackers has deep roots with developers, designers, makers, and other technical folks living in the Indianapolis metropolitan area. If you need a nerd, you need to post here.

Reply:
I am fan of @illbzo1's suggestion. What say _you_, @mileszs?